backend/infrahub/generators/tasks.py

```diff
@@ -178,6 +178,7 @@ async def request_generator_definition_run(
         definition__ids=[model.generator_definition.definition_id],
         include=["object"],
         branch=model.branch,
+        prefetch_relationships=True,
```

Comment:
Adding this line actually introduces these CI failures when executing generator's graphql query because `variables={"name": None}`. However, `graphql_query_post` payload does not accept non-null values for `variables` field, thus raising a 422 error. The root cause for `name` being `None` is that fetching relationships here overrides the `name` previously stored in the sdk store.